packages/core/src/external/carequality/ihe-gateway-v2/outbound/__tests__/process-dr.test.ts

```diff
@@ -173,7 +173,7 @@ describe("processDRResponse", () => {
       },
     });
 
-    expect(response?.operationOutcome?.issue[0]?.code).toBe("soap:Sender");
+    expect(response?.operationOutcome?.issue[0]?.code).toBe("schema-error");
```

Comment:
any particular reason why we use `toBe` here and `toEqual` in the same scenario in `process-dq.test`?